### ISSUE TITLE
[bug] Low resolution of notifications

### ISSUE THREAD

**Issue by life-the-user** [emersion/mako#430]
* What is the issue?

When spawning the first notification it has a very bad resolution.
After spawning second one, it fixes itself for some unknown reason.

* How to replicate?

Use sway, mako v1.6 and try spawning notifications using notify-send.

* How it supposed to work?

It supposed to have good resolution as it has after temporary fixing when spawning second notification.

My personal guess is that it has some problems with detecting screen resolution. When I also put cursor over it, the cursor becomes 3 times bigger until I move it away.
(no screenshot available yet, if required i will try to post it)

**Comment by emersion** [comment]
Can you reproduce with the latest commit?

**Comment by life-the-user** [comment]
No, it is definitely not about AppArmor. I will disable AppArmor and test, if it would change I will inform you.

**Comment by emersion** [comment]
I said nothing about AppArmor…

**Comment by fluix-dev** [comment]
> What was the latest commit?

By "Can you reproduce with the latest commit?" they just mean can you reproduce with the latest source code version as opposed to the release (which can be quite old). Nothing specific about the last commit :)

**Comment by diegov** [comment]
I ran into this today. It starts with the second notification, the first one renders at full resolution.

Here's an example:

```
$ killall mako
$ notify-send -a Notifications -c What Hello
```

At this point I get this:
![image](https://user-images.githubusercontent.com/297206/215766960-1f8a8cd3-f272-41a1-a05e-612bfe098af4.png)

If I run the same notify-send command again, I get this:

![image](https://user-images.githubusercontent.com/297206/215767040-11518834-6768-4a22-9b22-f1bd107a34a3.png)

I'm using the following config:
```
padding=4
margin=3
default-timeout=30000
border-radius=4
group-by=app-name,summary,category
anchor=top-right
font=Recursive Mn Lnr St Med 9
background-color=#25252FF5
progress-color=over #6F6F84F0
border-color=#202039E9
border-size=2
icons=1
format=<span foreground="#bcbcca">%a</span>\n%s\n%b
```

I tried removing the custom format and setting `markup=0` in mako's config, but I still see the same problem. First notification is fine, second one is blurry. 

This is on a 4k monitor, under swaywm, with `output "*" scale 2`.

The packages:
```
ii  mako-notifier  1.6-2        amd64        lightweight notification daemon for Wayland compositors
ii  sway           1.7-1        amd64        i3-compatible Wayland compositor
```

**Comment by tyalie** [comment]
Can confirm that I'm hitting upon the same issue. It's a bit unpredictable when it will render blurry and when not, but it's definitely blurry for the majority of the time.

What I observed thoo, is that if it is not blurry, it will pop-up blurry but then change almost immediately to being very sharp. It's easy to observe, as it actually triggers a slight layout change as the line spacing seemingly gets smaller just that tiny amount
